src/blocks/AdvanceContributors/render.php

```
@@ -0,0 +1,503 @@
+ * Render callback for the Contributors block
+ *
+ * @param array    $attributes Block attributes
+ * @param string   $content    Block content
+ * @param WP_Block $block      Block instance
+ * @return string
+ */
+// Prevent direct access
+if (!defined('ABSPATH')) {
+    exit;
+}
+function render_wedocs_advance_contributors_block($attributes, $content, $block) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$content'. (undefined)

(UnusedFormalParameter)